src/lua/luaTask.c

```diff
@@ -124,117 +146,206 @@ size_t get_ontick_freq()
     return 1000 / ticksToMs(onTickSleepInterval);
 }
 
-int getShouldReloadScript(void)
+void* getLua()
 {
-    return g_shouldReloadScript;
+    return g_lua;
 }
 
-void setShouldReloadScript(int reload)
+static void _terminate_lua()
```

Comment:
No.  I did this because there is already an extern method named `terminate_lua` that can get called at any time.  I needed a private internal version that was a subset of that public method (in this case one without locks).  Hence why I did that.  Its a common convention in the Linux kernel.  Usually it denotes that you "should really know what you are doing before you use it".  2 underscores exemplifies that even more.